lib/dialects/mariadb/query.js

```diff
@@ -59,11 +59,11 @@ class Query extends AbstractQuery {
       queryBegin = Date.now();
     } else {
       this.sequelize.log(
-        `Executing (${this.connection.uuid || 'default'}): ${this.sql}`,
+        `Executing (${this.connection.uuid || 'default'}): ${this.sql} with parameters ${JSON.stringify(parameters)}`,
```

Comment:
A few places where I have seen this now, we should probably use `Logger` class to inspect or probably use something central/configurable like `sequelize.inspect` IIRC it should already be there. My reasoning is, sometimes `parameters` are too big to log so users can override `sequelize.inspect` or other central inspect method, thus can make sense out of big block of text